p2p-specs/p2p-interface.md

```diff
@@ -572,18 +570,15 @@ The ENR MAY contain the following entries:
 
 Specifications of these parameters can be found in the [ENR Specification](http://eips.ethereum.org/EIPS/eip-778).
 
-#### Mempools bitfield
+#### Chain id field
 
-The ENR `mempools` entry signifies the mempools subnet bitfield with the following form
-to more easily discover peers participating in particular mempool id gossip subnets.
+ENRs MUST carry a `chain_id` key containing the ID of the network the bundler is connected to. This is done to ensure connections are made with peers on the intended Ethereum network.
 
-| Key                 | Value                                            |
-|:--------------------|:-------------------------------------------------|
-| `mempool_subnets`   | SSZ `Bitvector[MEMPOOL_ID_SUBNET_COUNT]`        |
+| Key        | Value        |
+|:-----------|:-------------|
+| `chain_id` | SSZ `uint64` |
```

Comment:
Some bundlers allow serving multiple chains in one instance (Skandha in particular). Can we make this value an array of `uint64`s if there are no downsides to it?

@dancoombs

Reply:
If a single instance wants to support multiple chain IDs I'd argue that it should use separate ports for each chain, and thus separate ENRs. Per chain configuration will be needed (bootnodes, mempool IDs).

The rest of the protocol doesn't include chain id in its messages, so for safety I think we should scope connections to a single chain id.

Would that work for Skandha?